chrome/skin/style-linux.css

```diff
@@ -0,0 +1,4 @@
+#universal-search-recommendation {
+  background-color: Highlight;
+  color: HighlightText;
+}/n
```

Comment:
@kaganjd Sorry, I didn't explain this correctly. I think you want to make sure your editor is inserting newlines at the end of files by default.

Here's a great article from Thoughtbot that explains the historical motivation for this convention, and how to configure common editors like vim and emacs: https://robots.thoughtbot.com/no-newline-at-end-of-file